apps/myaccount/src/api/change-password.ts

```diff
@@ -36,7 +36,8 @@ import { store } from "../store";
  * @param newPassword - newly assigned password.
  * @returns axiosResponse - a promise containing the response.
  */
-export const updatePassword = (currentPassword: string, newPassword: string): Promise<AxiosResponse> => {
+export const updatePassword = (currentPassword: string, newPassword: string,
+    isSubOrgUser: boolean = false, userOrganizationId: string = null): Promise<AxiosResponse> => {
```

Comment:
Will it be possible to retrieve the isSubOrg status inside the function itself using redux store instead of passing it as a param?